src/documents/models.py

```diff
@@ -483,8 +483,14 @@ def from_path(cls, path):
           "<title>.<suffix>"
         """
 
+        filename = os.path.basename(path)
+        for transform in settings.FILENAME_PARSE_TRANSFORMS:
+            (filename, count) = transform[0].subn(transform[1], filename)
```

Comment:
For clarity I'd suggest to destructure the `transform` tuple:

```python
for (pattern, repl) in settings.FILENAME_PARSE_TRANSFORMS:
    (filename, count) = pattern.subn(repl, filename)
```

Reply:
I like that better as well, thanks for the suggestion